sass/elements/_blocks.scss

```diff
@@ -15,3 +15,51 @@
 		}
 	}
 }
+
+hr {
+	border-bottom: 3px dotted $light-brown;
+
+	&.wp-block-separator {
+
+		&.is-style-dots {
+
+			&::before {
+				content: url("#{$sep-dots-sparkles}");
+				padding-left: 0;
+			}
+		}
+
+		&.is-style-default {
+			background: transparent;
+			border-style: dotted;
+			border-color: $light-brown;
+			border-image: url("#{$sep-dots}");
+			border-image-width: 4px;
+			border-image-slice: 2;
+			border-image-repeat: round;
+		}
+
+		&.is-style-wide {
```

Comment:
Maybe for this one, the sparkles are the `hr` background, and the borders on each side are `before`/`after`? (not totally sure it would work, but worth trying). The background might not always be the cream color, since we have color blocks on some screens.

Reply:
Yeah the background color was an issue. I couldn't think of a good way to create the space, but now that I've had a minute to let it site I could probably do a bg image for the sparkles and do a calc on the `before` & `after` widths. I'll give that a shot.

Reply:
Calc worked nicely on the `before` & `after`.